luigi/worker.py

```diff
@@ -971,10 +971,18 @@ def _handle_next_task(self):
                     self.add(t)
                 new_deps = [t.task_id for t in new_req]
 
+            try:
+                expl_json = json.dumps(expl)
+            except UnicodeDecodeError:
+                logger.info('Unicode decode error occur during %s task metadata serialisation, '
+                            'falling back to latin1 encoding.', task_id)
+
+                expl_json = json.dumps(expl, encoding='latin1')
```

Comment:
Ihor, this is a fallback, accidentally any dirt is possible in task metadata respecting any RFC or not, but this should not be the reason of DOS Luigi master.

json.dumps of the string = string, i don't clearly understand what other luigi contributors who made making expl handling are expecting so output processed exactly the same way as before except sustaining to the bad symbols and patch designed to be as non-invasive as possible.